common/src/main/java/io/netty/util/ResourceLeakDetector.java

```diff
@@ -588,10 +587,20 @@ public static void addExclusions(Class clz, String ... methodNames) {
         } while (!excludedMethods.compareAndSet(oldMethods, newMethods));
     }
 
-    private static final class TraceRecord extends Throwable {
+    private static class TraceRecord extends Throwable {
```

Comment:
Why is it not `final` anymore?

Reply:
@chrisvest because I extend it to override `fillInStacktrace(=`